docs/sources/alerting/unified-alerting/alerting-rules/_index.md

```diff
@@ -12,9 +12,9 @@ While queries and expressions select the data set to evaluate, a condition sets
 
 You can:
 
-- [Create Cortex or Loki managed alert rule]({{< relref "./create-cortex-loki-managed-rule.md" >}})
```

Comment:
I think this will break the link as we only rename the file reference and not the underlying file.

Reply:
On it.